docs/source/sci_api_tutorial/preparation_remove_hydrogens.rst

```diff
@@ -0,0 +1,150 @@
+==============================================
+ Preparation: Remove Hydrogens
+==============================================
+
+Briefs
+==============================================
+
+This science API, named ``enzy_htp.preparation.clean.remove_hydrogens``,
+removes hydrogen atoms from the supplied ``enzy_htp.structure.Structure`` class instance 
+(hereafter referred to as ``Structure`` instance).
+
+Input/Output
+==============================================
+
+**input**: A ``Structure`` instance (no matter it's a protein, polypeptite, or ligand).
+
+.. dropdown:: :fa:`eye,mr-1` How to obtain ``Structure`` instance?
+
+    Structure can be obtained by 
+    
+    1. parsing from a file using one of the StructureParser:
+
+    - `PDBParser <xxx>`_
+    - `PrmtopParser <xxx>`_
+
+    2. OR using the output of `Remove Solvent <preparation_remove_solvent.html>`_. (Commonly used here)
+
+**output**: A ``Structure`` instance of protonated structure (in-place modification, not as return value).
+
+.. panels::
+
+    :column: col-lg-12 col-md-12 col-sm-12 col-xs-12 p-2 text-left
+
+    .. image:: ../../figures/preparation_remove_hydrogens_dfd.svg
+        :width: 100%
```

Comment:
Please unify the format of the DFD. We only contain the i/o **data** here. Please remove the 2 config and the dotted arrows

Reply:
same for the protonation doc

Reply:
Resolved.